proxy/src/config.rs

```diff
@@ -194,6 +210,20 @@ const DEFAULT_PORTS_DISABLE_PROTOCOL_DETECTION: &[u16] = &[
 
 // ===== impl Config =====
 
+impl Config {
+    /// Modify a `trust-dns-resolver::config::ResolverOpts` to reflect
+    /// the configured minimum and maximum DNS TTL values.
+    pub fn configure_resolver_opts(&self, mut opts: ResolverOpts) -> ResolverOpts {
+        opts.positive_min_ttl = self.dns_min_ttl;
+        opts.positive_max_ttl = self.dns_max_ttl;
+        // TODO: Do we want to allow the positive and negative TTLs to be
+        //       configured separately?
```

Comment:
I don't know that we're eager to have them configured by end-users. However, we probably want to set good defaults for each. Since we're not setting default values for these (AFAICT) in this PR yet, we should file a follow-up issue to figure out good defaults.

Definitely I want to clamp the maximum negative TTL, for example.

Reply:
Sounds good, I'll do that.

Reply:
Opened #1054 to track this.